script/dev

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# This is a script for running Compose inside a Docker container. It's handy for
+# development.
+#
+# $ ln -s `pwd`/script/dev /usr/local/bin/docker-compose
+# $ cd /a/compose/project
+# $ docker-compose up
+# 
+
+set -e
+DIR="$(dirname "$(readlink "$0")")"/..
+docker build -t docker-compose $DIR
+exec docker run -i -t -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`:`pwd` -w `pwd` docker-compose $@
```

Comment:
Not sure what you mean? `docker run` will pick up `DOCKER_HOST`; the only assumption this line makes is that, wherever docker is running (i.e. on this machine, in a VM or in the cloud), it's also listening at `/var/run/docker.sock` so that can be mounted into the Compose container.

For Linux users and Mac folks using boot2docker, that's always going to be true. Arguably less certain in the case of remote servers and Swarm clusters, though.